src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -774,15 +782,22 @@ def visit_span(self, span: Span):
             return
 
         op = span.get("op", None)
-        allowed_span_ops = self.settings.get("allowed_span_ops")
-        if op not in allowed_span_ops:
+        if op not in ["resource.link", "resource.script"]:
```

Comment:
Hmm any particular reason why you removed the `allowed_span_ops` setting?

Reply:
I removed it because it wasn't configurable by flag, and didn't really feel like a setting - more like a part of the core logic of the detector. Different spans have different descriptions and properties - it isn't really something you can tweak without thinking about how the rest of the detector is going to react to it. That was my thinking anyway!